documentation/recommendations.neo4j-browser-guide

```diff
@@ -88,10 +88,10 @@
 </li>
 </ol>
 </div>
-<img src="http://guides.neo4j.com/sandbox/recommendations/img/title1.png" class="img-responsive">
+<img src="http://guides.neo4j.com/intro/img/title1.png" class="img-responsive">
```

Comment:
I think you forgot to pass the image base-url to run.sh, but I can do that before pushing to s3 :)

Just FYI

`neo4j_guides/run.sh recommendations.adoc recommendations.neo4j-browser-guide +1 http://guides.neo4j.com/sandbox/recommendations`